datafusion/physical-plan/src/coalesce_partitions.rs

```diff
@@ -92,10 +92,12 @@ impl DisplayAs for CoalescePartitionsExec {
                 }
                 None => write!(f, "CoalescePartitionsExec"),
             },
-            DisplayFormatType::TreeRender => {
-                // TODO: collect info
-                write!(f, "")
-            }
+            DisplayFormatType::TreeRender => match self.fetch {
+                Some(fetch) => {
+                    write!(f, "fetch: {fetch}")
```

Comment:
Maybe align with other operators?
```suggestion
                    write!(f, "limit: {fetch}")
```